frontend/src/metabase/ui/theme.ts

```diff
@@ -0,0 +1,43 @@
+import type { MantineThemeOverride } from "@mantine/core";
+
+import { color } from "metabase/lib/colors";
+
+export const theme: MantineThemeOverride = {
+  colors: {
+    brand: [color("brand-light"), color("brand")],
+    text: [color("text-light"), color("text-medium"), color("text-dark")],
+  },
+  primaryColor: "brand",
+  primaryShade: 1,
+  fontFamily: "Lato, sans-serif",
```

Comment:
Don't we have more fallback for Lato?
We are using `Lato, "Helvetica Neue", Helvetica, sans-serif` for www.